examples/SimpleRNN/main.jl

```diff
@@ -119,7 +119,7 @@ function compute_loss(x, y, model, ps, st)
     return binarycrossentropy(y_pred, y), y_pred, st
 end
 
-matches(y_pred, y_true) = sum((y_pred .> 0.5) .== y_true)
+matches(y_pred, y_true) = sum((y_pred .> 0.5f0) .== y_true)
```

Comment:
changed this for Metal compatibility, apple GPUs don't seem to support float64

Reply:
sounds good